test/parallel/test-http-common.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const httpCommon = require('_http_common');
+const checkIsHttpToken = httpCommon._checkIsHttpToken;
+const checkInvalidHeaderChar = httpCommon._checkInvalidHeaderChar;
+
+// checkishttptoken
+assert(checkIsHttpToken('t'));
+assert(checkIsHttpToken('tt'));
+assert(checkIsHttpToken('ttt'));
+assert(checkIsHttpToken('tttt'));
+assert(checkIsHttpToken('ttttt'));
+
+assert(!checkIsHttpToken(' '));
```

Comment:
Can you switch to `assert.strictEqual()`. It will make the code more readable, especially in the negated cases.